packages/core/test/mocks/client.ts

```diff
@@ -1,12 +1,22 @@
 import { Session } from '@sentry/hub';
-import { Event, Integration, Options, Severity, SeverityLevel, Transport } from '@sentry/types';
+import { ClientOptions, Event, Integration, Severity, SeverityLevel, Transport } from '@sentry/types';
 import { resolvedSyncPromise } from '@sentry/utils';
 
 import { BaseClient } from '../../src/baseclient';
 import { initAndBind } from '../../src/sdk';
 import { NewTransport } from '../../src/transports/base';
 import { NoopTransport } from '../../src/transports/noop';
-export interface TestOptions extends Options {
+
+export function getDefaultTestOptions(options: Partial<TestOptions> = {}): TestOptions {
+  return {
+    integrations: [],
+    transport: NoopTransport,
+    stackParser: () => [],
+    ...options,
+  };
+}
+
+export interface TestOptions extends ClientOptions {
```

Comment:
(optional) We may want to rename this interface to `TestClientOptions` to stay consistent with the new interface names. `TestOptions` is super ambiguous anyways.

^ would imply to also rename `getDefaultTestOptions` to `getDefaultTestClientOptions`.